eng/PatchConfig.props

```diff
@@ -60,4 +60,8 @@ Later on, this will be checked using this condition:
     <PackagesInPatch>
     </PackagesInPatch>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(VersionPrefix)' == '2.1.18' ">
```

Comment:
FYI it's fine to reuse an empty `<PropertyGroup>` for the previous servicing release. But, not worth further changes.